.ci/validation.sh

```diff
@@ -95,7 +95,7 @@ checkstyle-tester-diff-groovy-regression-single)
 
 codenarc)
   cd checkstyle-tester
-  ./codenarc.sh . diff.groovy > diff.log && cat diff.log && grep '(p1=0; p2=0; p3=0)' diff.log
+  ./codenarc.sh . diff.groovy > diff.log && cat diff.log && grep '(p1=0; p2=1; p3=0)' diff.log
```

Comment:
Can we avoid this ?

Reply:
> Number of parameters in method None.runCliExecution exceeds maximum allowed (5).

Not really. Command needs these number of parameters.

Reply:
lets follow limit. 
lets make method to be `def runCliExecution(allJar, mainClassOptions)`
where mainClassOptions if wrapper object with all data required.

Reply:
Should be done.